src/routes/api/decks/shared/+server.ts

```diff
@@ -20,8 +20,8 @@ export const POST = (async (event) => {
     ]);
 
   if (!deck) throw error(404, `Deck ${deckId} does not exist for user ${sourceUid}`);
-  if (!isCustomerSubscribedToBlendPro(sourceCustomer) && !isOrganizationMember(sourceUid)) throw error(401);
-  if (!isCustomerSubscribedToBlendPro(targetCustomer) && !isOrganizationMember(targetUid))
+  if (!isCustomerSubscribedToBlendPro(sourceCustomer) && !(await isOrganizationMember(sourceUid))) throw error(401);
+  if (!isCustomerSubscribedToBlendPro(targetCustomer) && !(await isOrganizationMember(targetUid)))
```

Comment:
Found a bug where this was always true because `isOrganizationMember` is a promise to that is always true, needed to await it

Reply:
Oh shit good catch!!